src/components/Searcher.js

```diff
@@ -719,8 +719,8 @@ class Searcher extends React.Component<SearcherDefaultProps, SearcherProps, Sear
   /**
    * Update the list of tags for the given document.
    */
-  updateTags(tags: Array<string>, docId: string): Promise<any> {
-    return this.search.updateRealtimeField(docId, FieldNames.TAGS, tags);
+  updateTags(tags: Array<string>, docId: string, onCompletion: () => void, onError: (error: string) => void): Promise<any> {
```

Comment:
I might be missing something, but I don't see a place where `updateTags` is used. This change isn't harmful, but I'd expect to see any place using `updateTags to be updated as well.

Reply:
nm, I see it being used through context and you already updated that.